performance/tests/OverallView.test.js

```diff
@@ -0,0 +1,186 @@
+const TestResults = require('../TestResults');
+const {CLS, FID, FPS, getAverageFPS, PageLoadingMetrics} = require('../TraceModel');
+const {clsValue, firstInputValue, getFileName} = require('../utils');
+
+describe('OverallView', () => {
+	const component = 'Overall';
+	TestResults.newFile(component);
+
+	it('FPS', async () => {
+		await FPS();
+		await page.goto('http://localhost:8080/overallView');
+		await page.waitForSelector('#virtualGridList');
+		await page.keyboard.down('ArrowUp');
+		await page.keyboard.up('ArrowUp');
+		await page.waitForTimeout(200);
+		await page.keyboard.down('ArrowUp');
+		await page.keyboard.up('ArrowUp');
+		await page.waitForTimeout(200);
+		await page.keyboard.down('ArrowUp');
+		await page.keyboard.up('ArrowUp');
+		await page.keyboard.down('ArrowDown');
+		await page.keyboard.up('ArrowDown');
+		await page.keyboard.down('ArrowDown');
+		await page.keyboard.up('ArrowDown');
+		await page.keyboard.down('ArrowDown');
+		await page.keyboard.up('ArrowDown');
+		await page.keyboard.down('ArrowRight');
+		await page.keyboard.up('ArrowRight');
+		await page.keyboard.up('ArrowDown');
+		await page.keyboard.down('ArrowDown');
+		await page.keyboard.up('ArrowDown');
+		await page.keyboard.down('ArrowDown');
+		await page.click('#tooltipButton'); // to move to the next panel.
+		await page.waitForTimeout(200);
+		await page.keyboard.down('ArrowDown');
+		await page.keyboard.up('ArrowDown');
+		await page.keyboard.down('ArrowDown');
+		await page.keyboard.up('ArrowDown');
+		await page.keyboard.down('ArrowDown');
+		await page.keyboard.up('ArrowDown');
+		await page.waitForTimeout(200);
```

Comment:
Why do we need a waitForTimeout for these parts?

Reply:
they are not needed